lib/ansiblelint/rules/MetaTagValidRule.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2018, Ansible Project
+
+from ansiblelint import AnsibleLintRule
+import re
+
+
+class MetaTagValidRule(AnsibleLintRule):
+    id = '702'
+    shortdesc = 'Tags must contain lowercase letters and digits only'
+    description = ("Tags must contain lowercase letters and digits only, "
+                   "and 'galaxy_tags' is expected to be a list")
+    tags = ['metadata']
+
+    TAG_REGEXP = re.compile('^[a-z0-9]+$')
```

Comment:
valid tag characters should include `_` (underscore) imho

Reply:
This is the current validation check that galaxy uses on import. If any edits are made in the galaxy importer, we'll update this rule so it matches.